base/server/src/main/java/com/netscape/cmscore/dbs/CertRecordPage.java

```diff
@@ -0,0 +1,77 @@
+// --- BEGIN COPYRIGHT BLOCK ---
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; version 2 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+//
+// (C) 2023 Red Hat, Inc.
+// All rights reserved.
+// --- END COPYRIGHT BLOCK ---
+package com.netscape.cmscore.dbs;
+
+import java.util.Iterator;
+import java.util.List;
+
+import com.netscape.certsrv.base.EBaseException;
+import com.netscape.certsrv.dbs.DBPagedSearch;
+
+/**
+* Contain all records in a page for a paged search.
+*
+* @author Marco Fargetta {@literal <mfargett@redhat.com>}
+*/
+public class CertRecordPage implements Iterable<CertRecord> {
```

Comment:
It is OK to rename the class to make more clear its behaviour. However, the classes `CertRecordList` and `CertRecordPage` have no common methods/interfaces so there is not need for the abstraction and since we have to drop the VLV I would not spend time to re-factoring the old code.